src/audio/host-zephyr.c

```diff
@@ -213,6 +214,15 @@ static int host_copy_one_shot(struct host_data *hd, struct comp_dev *dev, copy_c
 		return ret;
 	}
 
+	if (IS_ENABLED(CONFIG_DMA_NXP_SOF_HOST_DMA)) {
```

Comment:
I would say this is probably a common feature - i.e. we allocate a DMA channels and then should be able to use it for several one shot copies (with different SRC/DST addresses).
@kv2019i @lyakh @abonislawski can you see any problems if we always reload src/dst/size for every one shot copy ?

Reply:
Now this is performed for all platforms. Let's see if this breaks any tests. If not, we can probably keep it this way for now.